packages/transitive-overlay/src/util.ts

```diff
@@ -25,7 +125,7 @@ export function itineraryToTransitive(
     intl?: IntlShape;
   }
 ): TransitiveData {
-  const { isAccessMode, isFlex, isTransit } = coreUtils.itinerary;
+  console.log(itin);
```

Comment:
I think this was left in by mistake!

Reply:
Yes, reverted in https://github.com/opentripplanner/otp-ui/pull/452/commits/7ca81c36b3491b584b3136e005ab964e017e0a30.